run.ps1

```diff
@@ -0,0 +1,34 @@
+# set the base tools directory
+$toolsLocalPath = Join-Path (Get-Location) "Tools"
+$runExe = Join-Path $toolsLocalPath "Microsoft.DotNet.BuildTools.Run\run.exe"
+
+# if the tool doesn't exist then download and run the boot-strapper script
+if ((Test-Path $runExe) -eq 0)
+{
+    if ((Test-Path $toolsLocalPath) -eq 0)
+    {
+        mkdir $toolsLocalPath | Out-Null
+    }
+
+    # download boot-strapper script
+    $bootStrapperPath = Join-Path $toolsLocalPath "bootstrap.ps1"
+    Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/buildtools/master/bootstrap/bootstrap.ps1" -OutFile $bootStrapperPath
```

Comment:
It's conceivable that we won't always want to use master, however I would think that in this case run.ps1 would be updated directly (e.g. in foo/rel/1.1.0 they update to consume bootstrap.ps1 from buildtools/rel/1.1.0).